src/tqec/compile/observables/abstract_observable.py

```diff
@@ -46,13 +46,17 @@ class AbstractObservable:
         bottom_stabilizer_spatial_cubes: A set of spatial cubes of which
             the stabilizer measurements on the bottom face should be included in
             the observable.
+        temporal_hadamard_pipes: A set of pipes with the observable basis of which
+            a single stabilizer measurements at the realignment layer might be
+            included in the logical observable.
```

Comment:
Missing documentation about the second entry in the stored pairs. It seems to be the `Basis` at the minimum `z`, but it would be nice to have it explicitly written somewhere.